fbpic/lpa_utils/laser/antenna_injection.py

```diff
@@ -50,7 +53,8 @@ class LaserAntenna( object ):
     rest of the simulation uses cubic shape factors.)
     """
```

Comment:
Could you update the 3rd paragraph of the docstring of the `LaserAntenna` object above?
More specifically, we should specify that, when running on GPU:
- The update of the velocity of the virtual particles is either done on CPU or GPU, depending on whether the laser profile is GPU enabled.
- The deposition of charge/current is then always done on GPU.